app/app/settings.py

```diff
@@ -370,38 +392,19 @@
 }
 HOTJAR_CONFIG = {'hjid': env.int('HOTJAR_ID', default=0), 'hjsv': env.int('HOTJAR_SV', default=0), }
 
-# Rollbar - https://rollbar.com/docs/notifier/pyrollbar/#django
-ROLLBAR_CLIENT_TOKEN = env('ROLLBAR_CLIENT_TOKEN', default='')  # post_client_item
-ROLLBAR_SERVER_TOKEN = env('ROLLBAR_SERVER_TOKEN', default='')  # post_server_item
-if ROLLBAR_SERVER_TOKEN and ENV not in ['local', 'test']:
-    # Handle rollbar initialization.
-    ROLLBAR = {
-        'access_token': ROLLBAR_SERVER_TOKEN,
-        'environment': ENV,
-        'root': BASE_DIR,
-        'patch_debugview': False,  # Disable debug view patching.
-        'branch': 'master',
-        'exception_level_filters': [(Http404, 'ignored')],
-        'capture_ip': 'anonymize',
-        'capture_username': True,
-        'scrub_fields': [
-            'pw', 'passwd', 'password', 'secret', 'confirm_password', 'confirmPassword', 'password_confirmation',
-            'passwordConfirmation', 'access_token', 'accessToken', 'auth', 'authentication', 'github_access_token',
-            'github_client_secret', 'secret_key', 'twitter_access_token', 'twitter_access_secret',
-            'twitter_consumer_secret', 'mixpanel_token', 'slack_verification_token', 'redirect_state', 'slack_token',
-            'priv_key',
-        ],
+# Sentry
+SENTRY_USER = env('SENTRY_USER', default='')
+SENTRY_PASSWORD = env('SENTRY_PASSWORD', default='')
+SENTRY_ADDRESS = env('SENTRY_ADDRESS', default='')
+SENTRY_PROJECT = env('SENTRY_PROJECT', default='')
+SENTRY_RELEASE = raven.fetch_git_sha(os.path.abspath(os.pardir)) if SENTRY_USER else ''
+if SENTRY_ADDRESS and SENTRY_PROJECT:
+    RAVEN_CONFIG = {
+        'dsn': f'https://{SENTRY_USER}:{SENTRY_PASSWORD}@{SENTRY_ADDRESS}/{SENTRY_PROJECT}',
```

Comment:
We aren't using the hosted solution.  This gives whoever is running the app the choice of URI.  We're self-hosting, since it's a fairly small footprint and we can retain data as long as necessary.